common/src/main/java/com/wynntils/features/user/WynncraftPauseScreenFeature.java

```diff
@@ -23,48 +24,79 @@ public class WynncraftPauseScreenFeature extends UserFeature {
     @SubscribeEvent
     public void onPauseScreenInitEvent(PauseMenuInitEvent event) {
         PauseScreen pauseScreen = event.getPauseScreen();
-        List<Renderable> renderables = new ArrayList<>(pauseScreen.renderables);
+
+        Optional<Renderable> gridOpt = pauseScreen.renderables.stream()
+                .filter(x -> x instanceof GridWidget)
+                .findFirst();
+        if (gridOpt.isEmpty()) return;
+
+        GridWidget grid = (GridWidget) gridOpt.get();
```

Comment:
I realize I skipped past this file when reviewing earlier. This solution works, and if you don't want to muck around with it, I'm okay with that, but I just wanted to raise an idea: don't you think it would be easier to intercept the creation of the GridWidget instead?

Reply:
I think it's the same amount of "bad"

Reply:
Ok, let's keep this way then.